src/EnergyPlus/HeatBalanceHAMTManager.cc

```diff
@@ -1680,6 +1875,56 @@ namespace HeatBalanceHAMTManager {
         return WVDC;
     }
 
+    Real64 SatAbsHum(EnergyPlusData &state, Real64 const Temperature)
+    {
+        // FUNCTION INFORMATION:
+        //       AUTHOR         Florian Antretter
+        //       DATE WRITTEN   March 2018
+        //       MODIFIED       na
+        //       RE-ENGINEERED  na
+
+        // PURPOSE OF THIS FUNCTION:
+        // Compute Absolute Humidity in kg/m3 from Temperature
+
+        // METHODOLOGY EMPLOYED:
+        // na
+
+        // REFERENCES:
+        // Vaisala: Humidity conversion formulas (2013), eq (17)
+
+        // USE STATEMENTS:
+        // na
+
+        // Return value
+        Real64 SatAbsHum;
+
+        // Locals
+        // FUNCTION ARGUMENT DEFINITIONS:
+
+        // FUNCTION PARAMETER DEFINITIONS:
+        // na
+
+        // INTERFACE BLOCK SPECIFICATIONS:
+        // na
+
+        // DERIVED TYPE DEFINITIONS:
+        // na
+
+        // FUNCTION LOCAL VARIABLE DECLARATIONS:
+
+        Real64 VPSat;
+
+        VPSat = PsyPsatFnTemp(state, Temperature);
+
+        Real64 IdealGasConst;
+
+        IdealGasConst = 2.16679;
+
+        SatAbsHum = (IdealGasConst * VPSat) / (Temperature + 273.15) / 1000;
+
+        return SatAbsHum;
+    }
```

Comment:
Delete unused comments and collapse code (e.g. Real64 VPSat = ... , Real64 IdealGasConst = ... , and Return SatAbsHum = ... )